crates/bevy_ui/src/ui_node.rs

```diff
@@ -57,6 +71,70 @@ impl Default for Node {
     }
 }
 
+/// The node's position, rotation, and scaling.
+#[derive(Component, Debug, Clone, Deref, Reflect)]
+#[reflect(Component, Default)]
+pub struct NodeTransform(pub(crate) Affine2);
+
+impl NodeTransform {
+    pub const DEFAULT: Self = Self(Affine2::IDENTITY);
+}
+
+impl Default for NodeTransform {
+    fn default() -> Self {
+        Self::DEFAULT
+    }
+}
+
+/// Translation in logical pixels (before scaling) that is applied to the UI node and it's children
+#[derive(Component, Debug, Default, Clone, Deref, DerefMut, Reflect)]
+#[reflect(Component)]
+pub struct NodeTranslation(pub Vec2);
+
+impl NodeTranslation {
+    pub fn new(x: f32, y: f32) -> Self {
+        Self(Vec2::new(x, y))
+    }
+}
+
+/// Rotation in radians that is applied to the UI node and it's children
+#[derive(Component, Debug, Default, Clone, Deref, DerefMut, Reflect)]
+#[reflect(Component)]
+pub struct NodeRotation(pub f32);
+
+impl NodeRotation {
+    pub fn degrees(self) -> f32 {
+        self.0 * 180. / std::f32::consts::PI
+    }
+
+    pub fn from_degrees(degrees: f32) -> Self {
+        Self(degrees * std::f32::consts::PI / 180.)
+    }
+}
```

Comment:
those are not needed, they are part of std lib

https://doc.rust-lang.org/std/primitive.f32.html#method.to_degrees
https://doc.rust-lang.org/std/primitive.f32.html#method.to_radians